srf_generation/source_parameter_generation/generate_realisations_from_nhm.py

```diff
@@ -144,6 +144,10 @@ def generate_fault_realisations(
         realisation_file_name = get_srf_path(cybershake_root, realisation_name).replace(
             ".srf", ".csv"
         )
+
+        # if isfile(realisation_file_name):
```

Comment:
if you are leaving things commented in commits. Put an explanation of why with the code.